### DISCUSSION TITLE
Use with Unity3D

### DISCUSSION THREAD

**Discussion by lorenblue** [pamidur/aspect-injector#206] [General]
Hi there!

I am just now learning about aspects and such, so forgive me. Is there any way I can get this to work with Unity (the game engine)? I can add the nuget package just fine, and have no compiler errors, but when I try to copy the sample and call a decorated method, the logic in the attribute doesn't fire.

Cheers,

Lorenzo

**Comment by pamidur** [comment]
Hi @lorenblue , this is a good idea to make it work with Unity. In fact about a year ago I already tried to do it. I didn't manage to though I didn't invest much effort into it. I felt like there was no demand for this feature. I will look into Unity again soon. 

Cheers!

**Reply by lorenblue** [reply]
Hello, and sorry for never replying. I am still very interested! Cheers

**Comment by pamidur** [comment]
Hi Lorenzo,
I'm afraid this library does not work with Unity3D at the moment. There is
some work that needs to be done on our side to make it happen.

Please feel free to create an issue - feature request to support Unity3D

Cheers
